mk/distribution.mk

```diff
@@ -0,0 +1,86 @@
+PROJECT_NAME ?= kuma
```

Comment:
Remove one: https://github.com/kumahq/kuma/pull/6154 is merged as it introduced it too